test/functional/users_controller_test.rb

```diff
@@ -85,7 +85,7 @@ def setup
     # checks flash
     assert_equal flash[:error], I18n.t('users_controller.user_has_been_banned')
     # checks duplicated flash is not present
-    assert_equal flash[:notice], nil
+    assert_nil flash[:notice]
```

Comment:
Nice, this is really useful since that type of assertion will be deprecated soon in rails :+1:

Reply:
I've noticed that also, so I update it :rocket: Thanks @Uzay-G.